packages/flutter_tools/test/general.shard/devfs_test.dart

```diff
@@ -345,98 +295,6 @@ void main() {
   });
 }
 
-class MockVMService extends BasicMock implements VMService {
-  MockVMService() {
-    _vm = MockVM(this);
-  }
-
-  Uri _httpAddress;
-  HttpServer _server;
-  MockVM _vm;
-
-  @override
-  Uri get httpAddress => _httpAddress;
-
-  @override
-  VM get vm => _vm;
-
-  Future<void> setUp() async {
-    try {
-      _server = await HttpServer.bind(InternetAddress.loopbackIPv6, 0);
```

Comment:
I remove this functionality because we shouldn't be testing via a real server in a general shard test.